tasks/create_rhaptos_site.yml

```diff
@@ -42,3 +42,28 @@
 - name: start instance
   become: yes
   shell: supervisorctl restart plone-instance
+
+- name: send the script to create list of zope urls
+  when: rhaptos_site|failed
+  become: yes
+  template:
+    src: "misc/zope_urls_from_db.py"
```

Comment:
When I was writing this, I thought, this is only necessary because we have an existing database, so it shouldn't be in cnx-buildout. But yes, I think it makes sense to have it in cnx-buildout. I'll also try to redo the script so it doesn't use wget.